diff --git a/src/clang.rs b/src/clang.rs
index 3058448..01da383 100644
--- a/src/clang.rs
+++ b/src/clang.rs
@@ -188,17 +188,11 @@ impl Type {
     }
 
     pub fn size(&self) -> usize {
-        unsafe {
-            let val = clang_Type_getSizeOf(self.x);
-            if val < 0 { 0 } else { val as usize }
-        }
+        0
     }
 
     pub fn align(&self) -> usize {
-        unsafe {
-            let val = clang_Type_getAlignOf(self.x);
-            if val < 0 { 0 } else { val as usize }
-        }
+        0
     }
 
     // pointer
